### PR TITLE
master

### PR DESCRIPTION
**Pull request by sam65536**

### CODE REVIEW

File: homeworks/alexander.samiilenko_sam65536/homework_2/hometask2_2.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ */
+
+Main = (function () {
+    function Main() {
+    }
+
+    Main.reverse = function (number) {
+        if (number === 0) return 0;
+        let digits = (String(number).toString()).split('');
+        for (let i = 0; i < ((digits.length - 1) / 2 | 0); i++) {
```
Comment (markoffden) [inline homeworks/alexander.samiilenko_sam65536/homework_2/hometask2_2.js]: avoid using bitwise operators for such a simple task

File: homeworks/alexander.samiilenko_sam65536/homework_3/hometask3_1.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/**
+ * @return {number}
+ */
+function Sqr (number) {
```
Comment (markoffden) [inline homeworks/alexander.samiilenko_sam65536/homework_3/hometask3_1.js]: camelCase naming pls

File: homeworks/alexander.samiilenko_sam65536/homework_3/hometask3_1.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    return number * number;
+}
+
+function InitArr (step, max) {
+    let result = [];
+    let k = 0;
+    for (let i = 0; i <= max; i += step) {
+        result[k++] = i;
+    }
```
Comment (markoffden) [inline homeworks/alexander.samiilenko_sam65536/homework_3/hometask3_1.js]: pls refactor it to use push(), more readable solution

File: homeworks/alexander.samiilenko_sam65536/homework_3/hometask3_1.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    return result;
+}
+
+function SqrArr (arr) {
```
Comment (markoffden) [inline homeworks/alexander.samiilenko_sam65536/homework_3/hometask3_1.js]: naming again

**Comment by sam65536** [comment]
https://github.com/MastersAcademy/js-course-2018/pull/437

**Pull request closed without merging**